php5.6/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM php:5.6-apache
+MAINTAINER Derek P Sifford <dereksifford@gmail.com>
+
+# Install base requirements & sensible defaults + required PHP extensions
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+        less \
+        libpng12-dev \
+        libjpeg-dev \
+        mariadb-client \
```

Comment:
They're cross-compatible. `mariadb-client` is 100% compatible with MySQL and `mysql-client` is 100% compatible with MariaDB. Only one is necessary.